package.json

```diff
@@ -773,15 +773,15 @@
         "webpack-cli": "^4.6.0"
     },
     "dependencies": {
-        "@azure/arm-appservice": "^13.0.2",
+        "@azure/arm-appservice": "^15.0.0",
         "@azure/arm-resources": "^5.0.0",
         "@azure/core-client": "^1.7.3",
         "@azure/core-rest-pipeline": "^1.11.0",
-        "@microsoft/vscode-azext-azureappservice": "^3.1.0",
+        "@microsoft/vscode-azext-azureappservice": "file:../vscode-azuretools/appservice/microsoft-vscode-azext-azureappservice-3.3.1.tgz",
```

Comment:
I'll clean up the `package.json` and `package-lock.json` after these packages are ready to be imported through npm.